app/feature/feature-home/src/main/kotlin/com/hedvig/android/feature/home/home/ui/HomeDestination.kt

```diff
@@ -355,9 +393,65 @@ private fun HomeScreenSuccess(
   }
 }
 
+@Composable
+private fun ImportantMessages(
+  list: List<HomeData.VeryImportantMessage>,
+  openUrl: (String) -> Unit,
+  hideImportantMessage: (id: String) -> Unit,
+  modifier: Modifier = Modifier,
```

Comment:
This modifier now is not applied in all scenarios, and it's not applied to the top-most container composable inside this function.
AnimatedContent should be allowed to take a modifier, pass it there instead, and make sure to not use it again inside the Column